__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET:200 sends an array of topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics.length).toBe(3)
+        response.body.topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+        });
+      });
+  });
```

Comment:
Spot on first test and well done for adding the array length check here to make sure the response body absolutely contains some results.

Reply:
On the test description, trying using the term `responds with` rather than `sends`, as this makes it clearer that the test is checking the response (at the moment it reads as though you, as the client, are sending an array of topics to the request).